packages/ui-components/cypress/support/customPercyCommand.ts

```diff
@@ -212,7 +212,9 @@ export const installCustomPercyCommand = ({ before, elementOverrides }: {before?
 
     const titlePath = test.titlePath()
 
-    const screenshotName = name ? titlePath.concat(name).filter(Boolean).join(' > ') : ''
```

Comment:
Do we know why we needed to use `filter(Boolean)`? it suggests that we suspected the array to have falsy values. If that still holds then maybe we need to keep it??

Reply:
I have a gut feeling we don't need it, I took it out in the most recent change to test my hypothesis 🙂 we'll find out when the CI results come through!

Reply:
Confirmed, no impact to the screenshotName!

**How to test:**
1. In the code look for `cy.percySnapshot` where name is being passed in, i.e. `cy.percySnapshot('tall viewport')`
2. In this PR, search for the Percy snapshot with "tall viewport"
3. In another PR, [i.e. this one](https://percy.io/cypress-io/cypress/builds/18459177/search/preview/1035275312?browser=chrome&searchParam=short%20viewport&viewLayout=side-by-side&viewMode=new&width=1500), search for the Percy snapshot with "tall viewport"
4. Confirm that the snapshot names are the same

Reply:
Documenting a conversation @rachelruderman and I had about this:

I’m basically wondering if there’s a scenario where `test.titlePath()` may return an array with some items being empty/null. The answer turned out to be yes, if we elected to pass in an empty string in `describe` or `it`. Rachel suggested there is value in ending up with `something > > something` in this case as it will help draw attention to the undesirable use of an empty string, encouraging developers to use a more proper value.